specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHub-preview.json

```diff
@@ -1,1411 +1,1679 @@
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ipFilterRuleNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IpFilterRule"
+            },
+            "description": "The Namespace IpFilterRule."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Namespace IpFilterRule created",
+            "schema": {
+              "$ref": "#/definitions/IpFilterRule"
+            }
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Namespaces"
+        ],
+        "operationId": "Namespaces_DeleteIpFilterRule",
+        "x-ms-examples": {
+          "NameSpaceIpFilterRuleDelete": {
+            "$ref": "./examples/NameSpaces/IPFilterRule/EHNameSpaceIpFilterRuleDelete.json"
+          }
+        },
+        "description": "Deletes an IpFilterRule for a Namespace.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ipFilterRuleNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No content."
+          },
+          "200": {
+            "description": "Namespace IpFilterRule successfully deleted."
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Namespaces"
+        ],
+        "operationId": "Namespaces_GetIpFilterRule",
+        "x-ms-examples": {
+          "NameSpaceIpFilterRuleGet": {
+            "$ref": "./examples/NameSpaces/IPFilterRule/EHNameSpaceIpFilterRuleGet.json"
+          }
+        },
+        "description": "Gets an IpFilterRule for a Namespace by rule name.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/namespaceNameParamet
```

Comment:
NameSpaceVirtualNetworkRuleSetget -> NameSpaceVirtualNetworkRuleSetGet
Or change it to a sentence

Reply:
fixed, changed the example filenames from NameSpaceVirtualNetworkRuleSetget  to NameSpaceNetworkRuleSetGet